components/org.wso2.carbon.identity.remotefetch.core/src/main/java/org/wso2/carbon/identity/remotefetch/core/util/RemoteFetchConfigurationUtils.java

```diff
@@ -52,15 +56,19 @@ public static RemoteFetchCoreConfiguration parseConfiguration() throws RemoteFet
 
         boolean isEnabled = false;
         File workingDirectory = null;
-        String isEnabledProperty = IdentityUtil.getProperty("RemoteFetch.FetchEnabled");
-        String workingDirectoryProperty = IdentityUtil.getProperty("RemoteFetch.WorkingDirectory");
-        if (isEnabledProperty != null && !isEnabledProperty.isEmpty()) {
-            isEnabled = isEnabledProperty.equalsIgnoreCase("true");
-        }
-        if (workingDirectoryProperty != null && !workingDirectoryProperty.isEmpty()) {
+        String isEnabledProperty = IdentityUtil.getProperty(REMOTE_FETCH_ENABLED);
+        String workingDirectoryProperty = IdentityUtil.getProperty(REMOTE_FETCH_WORKING_DIRECTORY);
+
+        isEnabled = Boolean.parseBoolean(isEnabledProperty);
+        if (StringUtils.isNotBlank(workingDirectoryProperty)) {
             workingDirectory = new File(workingDirectoryProperty);
-            validateDirectory(workingDirectory);
+        } else {
+            // Set to default working directory at ${carbon.home}/tmp
+            String carbonHomeTmpDirPath = CarbonUtils.getCarbonHome() + File.separator + "tmp";
+            workingDirectory = new File(carbonHomeTmpDirPath);
         }
+        validateDirectory(workingDirectory);
+
```

Comment:
Can we remove this Extra Line?

Reply:
These spaces were intentionally kept for logical block separation and better readability.

Reply:
ACK